.github/workflows/web-backend.yml

```diff
@@ -0,0 +1,31 @@
+name: Build web backend
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: Build docker container
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        if: ${{ github.ref == 'refs/heads/master' }}
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build Docker image (and push on master)
+        uses: docker/build-push-action@v4.0.0
+        with:
+          push: ${{ github.ref == 'refs/heads/master' }}
+          tags: ghcr.io/cthit/goldapps-web-backend:latest
```

Comment:
Dont you want to push to docker.chalmers.it?

Reply:
Under our year as digIT we have primarily used ghcr as it is easier to use within GitHub Workflows and the images produced are listed in the repo next to the code.